### PR TITLE
turning in assignment

### PR DESCRIPTION
**Pull request by stephhappens**

### CODE REVIEW

**Review by rtablada — CHANGES_REQUESTED** [review]
Currently missing the sticky footer using display flex and flex-direction column.

File: public/index.html [hunk 1]
```diff
@@ -1,11 +1,83 @@
...
+  <title>Flex Practice</title>
   <link rel="stylesheet" href="/app.css">
 </head>
+
 <body>
+  <div class="full-page">
+    <nav class="top-nav">
+      <div class="nav-bar">
+        <ul class="nav-group">
+          <h1 class="nav-group nav-group__item">BACON SO HOT RIGHT NOW</h1>
```
Comment (rtablada) [inline public/index.html]: I usually will separate the grouping of left and right to make more sense semantically and to make life easier (this helps for flex or float based navbars).

File: public/index.html [hunk 2]
```diff
@@ -1,11 +1,83 @@
...
+      </div>
+    </nav>
+
+    <div class="hero">
+      <div class="hero__container">
+        <h1>
+          I AM HUNGRY FOR BACON!
+        </h1>
+      </div>
+      <div class="hero__container-absolute">
```
Comment (rtablada) [inline public/index.html:29]: To position the text within this centered using position: absolute, you will need a parent element that will be position relative and a child element that is position absolute.

File: public/index.html (hunk 1 above)
Comment (rtablada) [reply public/index.html]: Also, the navbar contents should be contained to a max-width for large screens

**Review by rtablada — CHANGES_REQUESTED** [review]
I need to pick back up and figure out other things to include with this review. But review the Rock Solid Layout docs in TIYO and that should help with the grid and containers.

**Comment by stephhappens** [comment]
- [x] Create a new project called `05 Flex Practice` using `yo sass-broccoli`
- [x] Use the `git` cli to initialize a new project
- [x] Use `hub create` to create a new repository on github called `02-box-model`
- [x] Use the `git` cli to create a branch called `develop`
- [x] Checkpoint: Change `title` for page
- [x] Use `hub pull-request -m "Turning in assignment"` to create a new pull request
- [x] Post link to pull request on online.theironyard.com
  - [ ] Checkpoint: Nav Bar
    - [ ] HTML
    - [ ] Styling
  - [ ] Checkpoint: Flexbox hero
    - [ ] HTML
    - [ ] Styling
  - [ ] Checkpoint: Position absolute hero
    - [ ] HTML
    - [ ] Styling
  - [ ] Checkpoint: Grid
    - [ ] HTML
    - [ ] Styling
  - [ ] Checkpoint: Card
    - [ ] HTML
    - [ ] Styling
  - [ ] Checkpoint: Footer
    - [ ] HTML
    - [ ] Styling

**Pull request merged**